### DISCUSSION TITLE
Cannot find module 'http'

### DISCUSSION THREAD

**Discussion by willsingh88** [DefinitelyTyped/DefinitelyTyped#63274] [Issues with a @types package]
On compile, started getting below error
```

Error: node_modules/@types/cors/index.d.ts:8:37 - error TS2307: Cannot find module 'http' or its corresponding type declarations.

 import { IncomingHttpHeaders } from 'http'; 
 
```
 
 **FYI:**  I have already tried the following : 

-  unistall/ install @types/cors
-  Rechecked  "compilerOptions": {
                                "typeRoots": ["node_modules/@types" ],
                                 "types": ["node"]

It suddenly started getting this error. No version update or git pull was done.

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Looks like it depends on `@types/node`

**Reply by Semigradsky** [reply]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63293
